src/Illuminate/Routing/Router.php

```diff
@@ -124,6 +125,13 @@ class Router implements BindingRegistrar, RegistrarContract
      */
     public static $verbs = ['GET', 'HEAD', 'POST', 'PUT', 'PATCH', 'DELETE', 'OPTIONS'];
 
+    /**
+     * Whether duplicate routes should be disallowed.
+     *
+     * @var bool
+     */
+    protected static $enforceUniqueRoutes = false;
```

Comment:
Don't think this should be static

Reply:
You're correct, it is a singleton. Thanks